pkg/virt-launcher/virtblocks/virtblocks.go

```diff
@@ -110,13 +110,17 @@ func (domain *VirtBlockDomainImpl) Create(dom *api.Domain) error {
 	} else if err != nil {
 		return fmt.Errorf("could not determine if process is already running: %v", err)
 	}
+	cmdStr = append(cmdStr, "-uuid", "272b47a8-b72b-4cd7-9109-79005816cc31")
```

Comment:
Can we just add UUID support to Virt Blocks instead of working around the missing feature in KubeVirt? Should be trivial to do so.

Reply:
Definitely. Just didn't get to it.